portfolio/src/main/webapp/style.css

```diff
@@ -57,7 +57,6 @@ body {
   padding-left: 20px;
   padding-right: 20px;
   padding-top: 200px;
-  padding-bottom: 20px;
 }
 
```

Comment:
Please order the declarations below.

Reply:
I removed padding-bottom so it is no longer there for this pull request. Is that what you were referring to or is there something else?

Reply:
Commented below, I meant the rest of the file below.

Reply:
Done!